src/views/ManageList.jsx

```diff
@@ -1,7 +1,63 @@
-export function ManageList() {
+import { useState } from 'react';
+import { addItem } from '../api';
+
+export function ManageList({ listPath }) {
+	const initialFormState = {
+		itemName: '',
+		daysUntilNextPurchase: 0,
+	};
+	const [newItem, setNewItem] = useState(initialFormState);
+
+	const handleChange = ({ target }) => {
+		setNewItem({ ...newItem, [target.name]: target.value });
+	};
+
+	const handleSubmit = async (event) => {
+		newItem.daysUntilNextPurchase = Number(newItem.daysUntilNextPurchase);
+		event.preventDefault();
+		try {
+			await addItem(listPath, newItem);
+			alert('Item successfully added!');
```

Comment:
This isn't blocking or anything, but it brings up a question Andrea and I were discussing yesterday about how to display a similar success/failure message (i.e., in an alert vs. in a DOM element) in our issue. We ended up going with a DOM element but we'll probably want to decide on a consistent approach across views eventually.

Reply:
This is a good thing to consider - I don't think that the alert way is any better in particular, and it might even slow down a user to have to dismiss a pop up window rather than simply have their view change when the updated data is available.

Reply:
https://www.npmjs.com/package/react-toastify

It might be something that you can consider for a custom alert in the future. Not necessary to implement in the project